tools/rpm2img

```diff
@@ -206,16 +210,34 @@ dd if="${BOTTLEROCKET_DATA}" of="${DATA_IMAGE}" conv=notrunc bs=1M seek=1
 sgdisk -v "${DISK_IMAGE}"
 sgdisk -v "${DATA_IMAGE}"
 
-lz4 -vc "${DISK_IMAGE}" >"${OUTPUT_DIR}/${DISK_IMAGE_NAME}"
-lz4 -vc "${DATA_IMAGE}" >"${OUTPUT_DIR}/${DATA_IMAGE_NAME}"
-lz4 -9vc "${BOOT_IMAGE}" >"${OUTPUT_DIR}/${BOOT_IMAGE_NAME}"
-lz4 -9vc "${VERITY_IMAGE}" >"${OUTPUT_DIR}/${VERITY_IMAGE_NAME}"
-lz4 -9vc "${ROOT_IMAGE}" >"${OUTPUT_DIR}/${ROOT_IMAGE_NAME}"
-chown 1000:1000 "${OUTPUT_DIR}/${DISK_IMAGE_NAME}" \
-    "${OUTPUT_DIR}/${DATA_IMAGE_NAME}" \
-    "${OUTPUT_DIR}/${BOOT_IMAGE_NAME}" \
-    "${OUTPUT_DIR}/${VERITY_IMAGE_NAME}" \
-    "${OUTPUT_DIR}/${ROOT_IMAGE_NAME}"
+if [[ ${OUTPUT_FMT} != "raw" ]]; then
+  QEMU_PARAMS="-f raw -O ${OUTPUT_FMT}"
+
+  # Stream optimization is required for creating an Open Virtual Appliance (OVA).
+  if [[ ${OUTPUT_FMT} == "vmdk" ]]; then
+    OUTPUT_OPTS="subformat=streamOptimized"
+  fi
+
+  if [[ ${OUTPUT_OPTS} != "none" ]]; then
+    QEMU_PARAMS="${QEMU_PARAMS} -o ${OUTPUT_OPTS}"
```

Comment:
I think this is a bit over complicated.  We can simplify it a bit by eliminating the multiple extra variables and having the qemu command hardcoded for now.  If you want to encapsulate all qemu args in a single variable that can be re-used, that's fine too.

IMHO it also might help to match on a positive case first to make it simpler to read:

```
if [[ ${OUTPUT_FMT} == "vmdk" ]]; then
    # Don't need to use this variable, could just hard code the commands
    QEMU_OPTS="convert -f raw -O vmdk -o subformat=streamOptimized"

    qemu-img "${QEMU_OPTS}" .....

# Can use elif's later for additional output formats
else
    ....
fi
```__

Reply:
I've updated my commits with something in-between your suggestion and the original commit.